src/menu/tooltipmenu.js

```diff
@@ -70,7 +70,7 @@ class TooltipMenu {
   }
 
   selectionCoords() {
-    let pos = this.config.position == "above" ? topCenterOfSelection() : bottomCenterOfSelection()
+    let pos = this.config.position == "above" ? topCenterOfSelection(this.pm.root) : bottomCenterOfSelection()
```

Comment:
Can this code be executed when `this.pm.root` is `null`? If so a null check may be worthwhile.